migrations/20200730_02_create_table_transactions.py

```diff
@@ -3,7 +3,7 @@
 __depends__ = {'20200730_01_create_table_users'}
 
 steps = [
-    step("CREATE TABLE transactions (id integer PRIMARY KEY,"
+    step("CREATE TABLE transactions (id uuid PRIMARY KEY,"
```

Comment:
In general the changes of migrations that are already in a working branch is not acceptable.
But I know the state of the project and that we don't have the app running anywhere (even on dev machines except mine and yours), so we can live with it.